src/Native/Unix/System.Native/pal_process.cpp

```diff
@@ -367,7 +367,11 @@ static int32_t ConvertRLimitResourcesPalToPlatform(RLimitResources value)
 static rlim_t ConvertFromManagedRLimitInfinityToPalIfNecessary(uint64_t value)
 {
     // rlim_t type can vary per platform, so we also treat anything outside its range as infinite.
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunknown-pragmas"
```

Comment:
I am not sure I understand the reason for ignoring the -Wunknown-pragmas here.

Reply:
`-Wunknown-pragmas` is for older Clang which don't have `-Wtautological-type-limit-compare`. It was failing on macOS and CentOS (Clang 3.9)

Reply:
Interesting, ok. I would expect that unknown pragma would warn against using a completely unknown pragma and not just a known pragma with unknown parameter.